metrics/web/lib/common/presentation/metrics_theme/model/dark_metrics_theme_data.dart

```diff
@@ -112,6 +113,22 @@ class DarkMetricsThemeData extends MetricsThemeData {
             successfulColor: ColorConfig.primaryColor,
             failedColor: ColorConfig.accentColor,
           ),
+          barGraphPopupThemeData: const BarGraphPopupThemeData(
+            color: Colors.white,
```

Comment:
It seems like the popup color for the dark theme is not white. 
![image](https://user-images.githubusercontent.com/70939496/94139515-6c9a4d80-fe72-11ea-9641-31283daab496.png)